CMakeLists.txt

```diff
@@ -513,17 +527,38 @@ else()
 endif()
 
 set(HEADERS_TO_CHECK
-    unistd.h io.h fcntl.h stdint.h
-    sys/time.h sys/types.h termios.h
-    values.h winsock.h sys/socket.h
-    dirent.h inttypes.h execinfo.h)
+    arpa/inet.h
+    dirent.h
+    endian.h
+    execinfo.h
+    fcntl.h
+    features.h
+    io.h
+    inttypes.h
+    linux/if.h
+    netinet/in.h
+    stdint.h
+    sys/ioctl.h
+    sys/select.h
+    sys/socket.h
+    sys/time.h
+    sys/types.h
+    termios.h
+    unistd.h
+    values.h
+    winsock.h
```

Comment:
Can you explain where all these extra headers came from?

Reply:
The additional headers are all either POSIX headers or glibc extensions. These aren't available on e.g. Windows. To see where they are used, you can search for them. For example, `arpa/inet.h` is used in `OOps/remote.c`